Examples for Geospatial schema files required by WorkRecordResource

donwilson-rezare opened the pull request:
I have created examples and validated them for all of the schema files required by the WorkRecordResource.

Commit:
Geospatial - schema file examples

Defined examples for some schema files, most of which are used in WorkRecordResource.

Commit:
Geospatial: Examples for schema required for WorkRecordResource

I have created examples and validated them for all of the schema files required by the WorkRecordResource.

cookeac commented on resources/WorkRecordResource.json:
```diff
@@ -1,44 +1,353 @@
+  },
+  "examples": [
+    {
+      "resourceType": "/crop/work-record",
```
Not sure that the ResourceType examples are correct. Normally they should reflect the Geospatial JSON file name. So WorkRecordResource, OperationRecordResource. Otherwise most code generated using openapi-generator will fail.

donwilson-rezare replied:
Changed Resource Types

cookeac commented:
Also this seems to introduce merge conflicts with PR #74

donwilson-rezare committed:
Update OperationRecordResource.json

Changed ResourceType

donwilson-rezare committed:
Update WorkRecordResource.json

Corrected resourceTypes.
Changed schemes for product registration and withdrawals

donwilson-rezare committed:
Update RegistrationType.json

Changed scheme

donwilson-rezare committed:
Merge branch 'master' into donwilson-gs-examples2

donwilson-rezare commented:
@cookeac I have altered the resource types.

I have set schemes for product registration to "nz.govt.nzfsa.eatsafe.acvm.register" and for withdrawals to "nz.govt.nzfsa.eatsafe.acvm.whp".  I don't know if these are appropriate.  I cannot find a list of WHP's on the ACVM website.
Also I have abbreviated the acvm URL somewhat.  Is that OK?

I don't know what to do for schemes that might be for commercial companies (e.g. fertiliser companies and testing laboratories).  I deliberately avoided using proper company names in case that caused issues.

I have resolved the conflict in lengthMeasureType.  I was improving the example.

cookeac commented:
> @cookeac I have altered the resource types.
> 
> I have set schemes for product registration to "nz.govt.nzfsa.eatsafe.acvm.register" and for withdrawals to "nz.govt.nzfsa.eatsafe.acvm.whp". I don't know if these are appropriate. I cannot find a list of WHP's on the ACVM website. Also I have abbreviated the acvm URL somewhat. Is that OK?

Yes, it is good where there are country-specific examples such as these to get them as close to reality as possible.
One piece of advice I was given by a leader in a government organisation was that organisational structures or names change regularly over time, while regulated systems and purposes evolve much more slowly. As the scheme URIs don't need to be URLs, you could consider "nz.govt.acvm.register", for instance.

> I don't know what to do for schemes that might be for commercial companies (e.g. fertiliser companies and testing laboratories). I deliberately avoided using proper company names in case that caused issues.

You're right not use an actual company name. Ideally over times these lists (e.g. tests) will be standardised across companies and use standard schemes. But at least some will remain company specific, and fictional data in examples is fine.
